Paymetheus/ViewModels/ImportScriptDialogViewModel.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) 2016 The btcsuite developers
+// Copyright (c) 2016 The Decred developers
+// Licensed under the ISC license.  See LICENSE file in the project root for full license information.
+
+using Grpc.Core;
+using Paymetheus.Decred.Util;
+using Paymetheus.Framework;
+using System;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Input;
+
+namespace Paymetheus.ViewModels
+{
+    public sealed class ImportScriptDialogViewModel : DialogViewModelBase
+    {
+        private DelegateCommandAsync _importScript;
+        public ICommand Execute => _importScript;
+
+        public ImportScriptDialogViewModel(ShellViewModel shell) : base(shell)
+        {
+            _importScript = new DelegateCommandAsync(ImportScriptAction);
+            _importScript.Executable = false;
+        }
+
+        private byte[] _scriptBytes;
+        public string ScriptHexString {
+            get { return Hexadecimal.Encode(_scriptBytes); }
+            set {
+                var scriptValid = false;
+                try
+                {
+                    _scriptBytes = Hexadecimal.Decode(value);
+                    scriptValid = true;
+                }
+                finally
+                {
+                    _importScript.Executable = scriptValid;
+                }
+            }        
+        }
+        public string Passphrase { private get; set; } = "";
+
+        private async Task ImportScriptAction()
+        {
+            try
+            {
+                await App.Current.Synchronizer.WalletRpcClient.ImportScriptAsync(_scriptBytes, true, 0, Passphrase);
```

Comment:
should rescan be exposed to the UI?

This also needs more testing.  last time i imported a script, i closed pm while it was still rescanning.  next startup, wallet marked itself in sync with the last block that it rescanned from the script import, and i could not open paymetheus at all (needed to run dcrwallet manually in order to get the wallet synced again).